app/views/admin/_adnav.html.erb

```diff
@@ -1,9 +1,9 @@
  <ul class="nav nav-tabs">
-    <li role="presentation" class=<%= 'active' if request.fullpath == dashboard_path %>><%= link_to 'Admin Dashboard', dashboard_path %></li>
-    <li role="presentation" class=<%= 'active' if request.fullpath == new_room_path %>><a href="/rooms/new">Create a Room</a></li>
-    <li role="presentation" class=<%= 'active' if request.fullpath == admin_reports_meetings_path %>><a href="/admin/reports_meetings">Meetings Report</a></li>
-    <li role="presentation" class=<%= 'active' if request.fullpath == admin_reports_rooms_path %>><a href="/admin/reports_rooms">Rooms Report</a></li>
-    <li role="presentation" class=<%= 'active' if request.fullpath == admin_busiest_employees_path %>><a href="/admin/busiest_employees">Employee Report</a></li>
-    <li role="presentation" class=<%= 'active' if request.fullpath == admin_add_branding_path %>><a href="/admin/add_branding">Branding</a></li>
-    <li role="presentation" class=<%= 'active' if request.fullpath == admin_invitations_show_path %>><a href="/admin/invitations/show">Meeting invites</a></li>
+    <li role="presentation" class=<%= add_class_if(dashboard_path,'active') %>><%= link_to 'Admin Dashboard', dashboard_path %></li>
+    <li role="presentation" class=<%= add_class_if(new_room_path, 'active') %>><a href="/rooms/new">Create a Room</a></li>
+    <li role="presentation" class=<%= add_class_if(admin_reports_meetings_path, 'active') %>><a href="/admin/reports_meetings">Meetings Report</a></li>
+    <li role="presentation" class=<%= add_class_if(admin_reports_rooms_path, 'active') %>><a href="/admin/reports_rooms">Rooms Report</a></li>
+    <li role="presentation" class=<%= add_class_if(admin_busiest_employees_path, 'active') %>><a href="/admin/busiest_employees">Employee Report</a></li>
+    <li role="presentation" class=<%= add_class_if(admin_add_branding_path, 'active') %>><a href="/admin/add_branding">Branding</a></li>
+    <li role="presentation" class=<%= add_class_if(admin_invitations_show_path, 'active') %>><a href="/admin/invitations/show">Meeting invites</a></li>
```

Comment:
What if you made a helper method that wraps some of the redundancy here?

```rb
def active_if(path)
  add_class_if(path, 'active')
end
```

So then all the links become:
```erb
<li role="presentation" class=<%= active_if(dashboard_path,'active') %>><%= link_to 'Admin Dashboard', dashboard_path %></li>
```